deb/publish.go

```diff
@@ -452,6 +453,36 @@ func (p *PublishedRepo) GetLabel() string {
 	return p.Label
 }
 
+// GetAppStream returns full path list of appstream files
+func (p *PublishedRepo) GetAppStream(component string) ([]string, error) {
+	if utils.Config.AppStreamDir == "" {
+		return []string{}, nil
```

Comment:
return a custom error instead to inform the caller that this operation didn't occur